src/SwedbankPay.Sdk.Tests/TestHelpers/TestHelper.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Microsoft.Extensions.Configuration;
+
+namespace SwedbankPay.Sdk.Tests.TestHelpers
+{
+    public class TestHelper
+    {
+
+        public static IConfigurationRoot GetIConfigurationRoot(string outputPath)
+        {
+#if DEBUG
```

Comment:
Consider removing this if-else as it can lead to unexpected behavior when using the SDK package. Just imagine what would happen if a project using the SDK is run locally in debug mode, for example.

Reply:
This is the Test project, this have nothing to do with how the SDK is run.